web3c/oasis.js

```diff
@@ -169,10 +169,23 @@ class Oasis {
   }
 
   _setupSubscribe(options) {
-    this.subscribe = (subscription, subscribeOptions, callback) => {
+    this.subscribe = function() {
```

Comment:
you can use named arguments in combination with the `arguments` array

Reply:
I wanted to avoid confusion to make sure that arguments always come from the same place, and we don't start using `filter` or `type` from the function arguments instead of from the `arguments` array